<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgSeq</class>
 <widget class="QMainWindow" name="MsgSeq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send sequence...</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addMsgButton">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Add Message</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendAllButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Send All</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendSelectedButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Send Selected</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeAllButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Remove All</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSelectedButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Selected</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QTableView" name="msgTableView">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="saveMsgSeq"/>
    <addaction name="openMsgSeq"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMessage">
    <property name="title">
     <string>&amp;Message</string>
    </property>
    <addaction name="actionRemoveSelected"/>
    <addaction name="actionRemoveAll"/>
    <addaction name="separator"/>
    <addaction name="actionSendSelected"/>
    <addaction name="actionSendAll"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuMessage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveMsgSeq">
   <property name="text">
    <string>&amp;Save message sequence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="openMsgSeq">
   <property name="text">
    <string>&amp;Open message sequence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRemoveSelected">
   <property name="text">
    <string>&amp;Remove selected</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="text">
    <string>R&amp;emove all</string>
   </property>
  </action>
  <action name="actionSendSelected">
   <property name="text">
    <string>&amp;Send selected</string>
   </property>
  </action>
  <action name="actionSendAll">
   <property name="text">
    <string>Se&amp;nd all</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
